### PR TITLE
Allow configuration of useAnchor parameter for linker

### PR DESCRIPTION
**Pull request by alex-deaconu**
We need to use `true` for the RVA. Setting `false` for Apps to enable help center link anchors.

Staged at https://apps-stage-1.risevision.com

@Rise-Vision/apps please review. Thanks!

### CODE REVIEW

File: dist/js/components/segment-analytics.js [hunk 1]
```diff
@@ -5,9 +5,11 @@
   angular.module("risevision.common.components.analytics", [])
 
   .value("SEGMENT_API_KEY", "AFtY3tN10BQj6RbnfpDDp9Hx8N1modKN")
+    .value("GA_LINKER_USER_ANCHOR", true)
```
Comment (ezequielc) [inline dist/js/components/segment-analytics.js]: @alex-deaconu I think it should be spelled GA_LINKER_USE_ANCHOR (USE instead of USER), right?

Comment (alex-deaconu) [reply dist/js/components/segment-analytics.js]: @ezequielc you're right, will fix before merging. Thanks!

**Comment by alex-deaconu** [comment]
@ezequielc that is correct re: PR for RVA. Ideally we would default to `false` and set `true` in RVA, but I think this is ok too. Thoughts? Thanks.

**Comment by ezequielc** [comment]
@alex-deaconu This is good with me. 👍 Thanks!

**Pull request merged**